.taplo.toml

```diff
@@ -0,0 +1,22 @@
+[formatting]
+align_comments = false
+# Align consecutive entries vertically.
+align_entries = true
+# Maximum amount of allowed consecutive blank lines. This does not affect the whitespace at the end of the document, as it is always stripped.
+allowed_blank_lines = 1
+# Collapse arrays that don't exceed the maximum column width and don't contain comments.
```

Comment:
I'm not familiar with this taplo tool, but it seems nice. Should we start using it consistently through the codebase?